package.json

```diff
@@ -8,6 +8,7 @@
     "@testing-library/jest-dom": "^5.14.1",
     "@testing-library/react": "^13.0.0",
     "@testing-library/user-event": "^13.2.1",
+    "axios": "^1.5.0",
```

Comment:
do you know what `ˆ` means on package.json? 👀 
if not, i do recommend [reading about it](https://github.com/isaacs/node-semver#caret-ranges-123-025-004), so you know why in centauro we always keep the version without it

Reply:
Right, I'll read about it, thanks for the tip